init.zsh

```diff
@@ -7,8 +7,12 @@ if [[ -z "$ANTIGEN_HS_MY" ]]; then
     ANTIGEN_HS_MY="$ANTIGEN_HS_HOME/../MyAntigen.hs"
 fi
 
+antigen-hs-init () {
+  cd "$ANTIGEN_HS_HOME" && stack build
+}
+
 antigen-hs-compile () {
-  runghc -i"$ANTIGEN_HS_HOME/" -- "$ANTIGEN_HS_MY"
+  stack exec -- runghc -i"$ANTIGEN_HS_HOME/" -- "$ANTIGEN_HS_MY"
```

Comment:
What about keeping the runghc way by default, but also have an option which, if enabled, uses `cabal install` instead so that nix/guix people can use that to build it?

But I mean, non-haskellers can still use nix/guix themselves to manage haskell packages. nix/guix does can be used in any distribution, not just in nixos/guixsd.